server_config.toml

```diff
@@ -150,7 +150,7 @@ listener = "tcp://0.0.0.0:46658"
 # This file needs to be in the root directory
 private_validator_file = "priv_validator.json"
 
-  # Tendermint requires additional configuration parameters.
+  # Tendermint requires additional configuration parameters.TransactAndHold
```

Comment:
oops yeah, I'll fix that. I more or less agree, at least not in this form. Having send that i think there is an argument to have the config templating as a part of eris-db/eris-client, so changes to config happen in the same place as they are consumed. i.e. move some of what is in eris-cm. But let's shelf this.